### PR TITLE
Merge Change from Upstream (Boxen/puppet-nginx)

### PR DESCRIPTION
**Pull request by coreyja**
/domain @while1malloc0

### CODE REVIEW

File: files/brews/nginx.rb [hunk 1]
```diff
@@ -32,21 +39,26 @@ def passenger_config_args
...
             "--pid-path=/opt/boxen/data/nginx/nginx.pid",
             "--lock-path=/opt/boxen/data/nginx/nginx.lock"]
 
-    args << passenger_config_args if build.with? "passenger"
-    args << "--with-http_dav_module" if build.with? "webdav"
+    args << passenger_config_args if ARGV.include? '--with-passenger'
+    args << "--with-http_dav_module" if ARGV.include? '--with-webdav'
+    args << "--with-http_gzip_static_module" if ARGV.include? '--with-gzip-static'
+    args << "--with-http_v2_module" if ARGV.include? "--with-http2"
     args << "--with-debug" if build.with? "debug"
```
Comment (coreyja) [inline files/brews/nginx.rb]: This is the only reason that we are on a private fork. We want this debug option that isn't in the boxen upstream module. Might try to get a patch into the upstream eventually. But for now keeping a fork is fine!

**Comment by coreyja** [comment]
This will be consumed when we merge the Boxen upstream changes. They upgraded the version of the nginx module they use, so I wanted to get our fork up to date as well.

**Comment by while1malloc0** [comment]
>> domain lgtm

**Comment by while1malloc0** [comment]
<< domain lgtm

**Pull request merged**